modules/gcp-stackdriver-export/main.tf

```diff
@@ -0,0 +1,54 @@
+# ------------------------------------------------------------------------------
+# TERRAFORM / PROVIDER CONFIG
+# ------------------------------------------------------------------------------
+
+terraform {
+  # The configuration for this backend will be filled in by Terragrunt
+  backend "gcs" {}
+}
+
+provider "google" {
+  project     = "${var.project_id}"
+  credentials = "${var.serviceaccount_key}"
+}
+
+# ------------------------------------------------------------------------------
+# GOOGLE STACKDRIVER EXPORT
+# ------------------------------------------------------------------------------
+
+resource "google_storage_bucket" "exported-logs" {
+  name = "${var.project_id}-exported-logs"
+}
+
+resource "google_logging_project_sink" "my-export" {
+  count       = "${length(var.exports)}"
+  name        = "${element(keys(var.exports), count.index)}"
+  destination = "storage.googleapis.com/${google_storage_bucket.exported-logs.name}"
+
+  # logName is a common thing to filter on. Per
+  # https://cloud.google.com/logging/docs/view/advanced-filters#minimize_global_and_substring_searches,
+  # it is best to match on an exact string. Doing so requires knowing
+  # var.project_id (since logName starts with "projects/my-project-id/").
+  # Neither Terraform variables (var.project_id) nor their environment variable
+  # cousins (TF_VAR_project_id) are availble from terraform.tfvars, where an
+  # instance of this module declares its filters. Hence, this workaround to
+  # replace the string "__var.project_id__" with the value of ${var.project_id}
+  # at runtime.
+  filter = "${replace(
```

Comment:
I would again question the necessity of the filter, as it looks like we have `project_id` available when  using this module, but again it seems like it shouldn't break anything.

Reply:
I would again ask you to propose a better fix.

Reply:
This discussion is continuing in https://github.com/gpii-ops/exekube/pull/12.